app/plugins/okta-custom-provider.js

```diff
@@ -0,0 +1,42 @@
+// this is work done by Tim Costa (https://github.com/timcosta)
```

Comment:
I think it's worth either tidying up this or wrapping it in comments which make it explicit that this is a cut and paste from the PR and doesn't conform to our standards. If it's not a cut and paste then we should change it to use `const` and string interpolation.